google/cloud/bigtable/client.py

```diff
@@ -170,6 +171,12 @@ def __init__(
         self._client_info = client_info
         self._emulator_host = os.getenv(BIGTABLE_EMULATOR)
 
+        if self._emulator_host is not None:
+            if credentials is None:
+                credentials = AnonymousCredentials()
+            if project is None:
+                project = "nonesuch-project-123"
+
```

Comment:
Avoid accidentally falling back to `gcloud` or other environmental credentials when the emulator host environment variable is set.

Reply:
similar feedback to firestore review, can we use a project value here that can't exist? google is a reserve word, so that may work?
https://github.com/googleapis/python-firestore/pull/464#discussion_r718872235

Reply:
0500e19